components/VM_Arm/src/modules/init_ram.c

```diff
@@ -17,14 +17,7 @@ extern unsigned long ram_size;
 
 void WEAK init_ram_module(vm_t *vm, void *cookie)
 {
-    int err;
-
-    if (config_set(CONFIG_PLAT_EXYNOS5) || config_set(CONFIG_PLAT_QEMU_ARM_VIRT) || config_set(CONFIG_PLAT_TX2)) {
-        err = vm_ram_register_at(vm, ram_base, ram_size, true);
-    } else {
-        err = vm_ram_register_at(vm, ram_base, ram_size, false);
-    }
-    assert(!err);
+    assert(!vm_ram_register_at(vm, ram_base, ram_size, vm->mem.unity));
```

Comment:
Under Release builds, this assert would be removed by the compiler, and the call to vm_ram_register_at would be removed also.

Reply:
Good to know. Will update.